src/game/editor/editor.cpp

```diff
@@ -592,6 +594,26 @@ void CEditor::CallbackSaveMap(const char *pFileName, int StorageType, void *pUse
 	pEditor->m_Dialog = DIALOG_NONE;
 }
 
+void CEditor::CallbackOpenEntities(const char *pFileName, int StorageType, void *pUser)
+{
+	CEditor *pEditor = (CEditor*)pUser;
+	CImageInfo ImgInfo;
+	if(!pEditor->Graphics()->LoadPNG(&ImgInfo, pFileName, StorageType))
+		return;
+	
+	if((ImgInfo.m_Width % 16) || (ImgInfo.m_Height % 16))
```

Comment:
You can remove these brackets
```suggestion
	if(ImgInfo.m_Width % 16 || ImgInfo.m_Height % 16)
```